docs/command-line-interface/aserto-idp-cli/usage.mdx

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: Usage
+---
+
+# Usage
+
+## Importing user data
+
+To import user data from an IDP to aserto:
+
+```
+aserto-idp exec --from json --to aserto -c PATH_TO_CONFIG
+```
+
+Note that if json or aserto plugins are not on the system, using this command, they will be automatically downloaded. Also if there is a newer version of either one of the plugins used, the following message will be prompted:
+
+```
+A new version '0.0.12' of the plugin 'json' is available
+```
+
+To disable updates checking when using `exec` or `delete` :
+
+```
+aserto-idp exec --from json --to aserto -c PATH_TO_CONFIG --no-update-check
+```
+
+or
+
+```
+aserto-idp exec --from json --to aserto -c PATH_TO_CONFIG -n
+```
+
```

Comment:
Another scenario that would be interesting to show here is "Creating a dry-run of an IDP import"

Something like:

## Creating a dry-run of an IDP import

To generate a JSON file containing all the users that would be imported from your IDP, use a command like the following:

```
aserto-idp exec --from auth0 --to json -c PATH_TO_CONFIG
```